tests/integration/ovs_test.py

```diff
@@ -103,3 +117,44 @@ def test_create_and_remove_ovs_bridge_with_internal_port_and_static_ip():
 
     assertlib.assert_absent(BRIDGE1)
     assertlib.assert_absent(PORT1)
+
+
+@pytest.mark.xfail(raises=NmstateLibnmError, strict=True)
+def test_nm_ovs_plugin_missing():
+    with disable_nm_ovs_plugin():
+        libnmstate.apply(
+            {
+                Interface.KEY: [
+                    {
+                        Interface.NAME: BRIDGE1,
+                        Interface.TYPE: InterfaceType.OVS_BRIDGE,
+                        Interface.STATE: InterfaceState.UP,
+                    }
+                ]
+            }
+        )
+
+
+@contextmanager
+def disable_nm_ovs_plugin():
+    libcmd.exec_cmd(DNF_REMOVE_NM_OVS_CMD)
+    libcmd.exec_cmd(SYSTEMCTL_RESTART_NM_CMD)
+    libnmstate.nm.checkpoint._nmdbus_manager = None
+    ret = [1, 0, 0]
+    elapsed_time = 0
+    while ret[0] != 0 and elapsed_time < 20:
```

Comment:
Initially, the hope was that we needed this to get the error fixed but actually we needed to clear the dBus manager. Is this loop still needed? If yes, it would be great to make it more generic as in https://github.com/nmstate/nmstate/blob/ca3450dc93a4dc3af9764c832214c2c250a92535/tests/integration/dynamic_ip_test.py#L688 to avoid repeating it twice here.

Reply:
This is not needed anymore.